### PR TITLE
[FIX] OWGeocoding: data concatenation error

### PR DESCRIPTION
**Pull request by robertcv**
##### Issue
Fixes crash because of deprecated Table.concatenate for columns.
##### Description of changes
Replace it with Table.transform

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ cache:
 matrix:
   include:
     - python: '3.6'
-      env: ORANGE="3.16.0"
+      env: ORANGE="3.20.0"
```
Comment (janezd) [inline .travis.yml]: Wouldn't it be better to test against the latest version? `Table.concatenate` is changed in 3.24; 3.20 wouldn't require the below changes, AFAIK:

Comment (robertcv) [reply .travis.yml]: The matrix includes also the latest release and master.

**Comment by robertcv** [comment]
@janezd could you pleas merge this

**Comment by ajdapretnar** [comment]
The error that appears here is the same as the one reported in #80.

**Comment by ajdapretnar** [comment]
Fixes #81.

**Comment by robertcv** [comment]
@ajdapretnar The TimeoutError is not connected to this pr. Can you or @janezd look at this and merge this.

**Comment by ajdapretnar** [comment]
> The TimeoutError is not connected to this pr.

I never said it is. 😄 It is just the same error that same error that appeared yesterday at lectures and I couldn't reproduce it. Now it was reproduced in the tests, so if anyone has any idea what the error means, (s)he should add explanation to #80.

**Pull request merged**